src/genpy/generate_struct.py

```diff
@@ -165,12 +165,12 @@ def unpack(var, pattern, buff):
     return var + ' = _get_struct_%s().unpack(%s)' % (pattern, buff)
 
 
-def unpack2(var, pattern, buff):
```

Comment:
I'm unsure if should rather avoid this change in order to maintain API/ABI compatibility. On the other hand this function is very likely just used within genpy

Reply:
Making the change API compatible if that is easily possible is certainly the better approach since you simply don't know which public API is used somewhere.

Reply:
I introduced unpack3, leaving unpack2 unchanged but also uncalled from within genpy.